pkg/controller/datavolume/controller-base.go

```diff
@@ -1192,3 +1249,13 @@ func (r *ReconcilerBase) shouldUseCDIPopulator(syncState *dvSyncState) (bool, er
 
 	return usePopulator, nil
 }
+
+func (r *ReconcilerBase) pvcRequiresNoWork(pvc *corev1.PersistentVolumeClaim, dv *cdiv1.DataVolume) (bool, error) {
```

Comment:
Can we flip the name and logic for this to `pvcRequiresWork` I am seeing a lot of double negatives in the rest of the code `if !pvcRequiresNoWork` which means `if pvcRequiresWork`. This would make it easier to read and understand.

Reply:
Changed to `pvcRequiresWork`